datagen/README.md

```diff
@@ -27,13 +26,13 @@ If you are building with a jdk version that is not 8, you will need to add in th
 corresponding profile flag `-P<jdk11|jdk17>`
 
 After this the jar should be at 
-`target/rapids-4-spark-integration-tests_2.12-$PLUGIN_VERSION-spark$SPARK_VERSION.jar`
+`target/datagen_2.12-$PLUGIN_VERSION-spark$SPARK_VERSION.jar`
```

Comment:
should this jar be something like `rapids-4-spark-datagen`? I guess we are not publishing it, so maybe that's not such a big deal (feels a bit generic)

Reply:
I originally had it named that way, but there's nothing in it specific to the RAPIDS Accelerator for Apache Spark.  It only uses Apache Spark interfaces and doesn't require the RAPIDS Accelerator.